specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/virtualWan.json

```diff
@@ -0,0 +1,2129 @@
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of all the VirtualWANs in the subscription.",
+            "schema": {
+              "$ref": "#/definitions/ListVirtualWANsResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnSites/{vpnSiteName}": {
+      "get": {
+        "operationId": "VpnSites_Get",
+        "x-ms-examples": {
+          "VpnSiteGet": { "$ref": "./examples/VpnSiteGet.json" }
+        },
+        "description": "Retrieves the details of a VPNsite.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VpnSite."
+          },
+          {
+            "name": "vpnSiteName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VpnSite being retrieved."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of the VpnSite retrieved.",
+            "schema": {
+              "$ref": "#/definitions/VpnSite"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "VpnSites_CreateOrUpdate",
+        "x-ms-examples": {
+          "VpnSiteCreate": { "$ref": "./examples/VpnSitePut.json" }
+        },
+        "description": "Creates a VpnSite resource if it doesn't exist else updates the existing VpnSite.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VpnSite."
+          },
+          {
+            "name": "vpnSiteName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VpnSite being created or updated."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "VpnSiteParameters",
```

Comment:
Some of your parameter names are camel cased, some are pascal cased. We recommend having the parameter names camel cased.

Reply:
Leaving them as is.

Reply:
Sure - but in the case of VpnConnections_CreateOrUpdate it's mismatching - the parameter name is vpnConnectionParameters but VpnConnectionParameters in the example

Reply:
Sure- fixing that. Also, as per the suggestion by @MikhailTryakhov changing the Api version to 2018-04-01 for this swagger and hence moving everything to 2018-04-01 folder.